starknet_devnet/blueprints/base.py

```diff
@@ -126,25 +126,33 @@ def load():
 
 
 @base.route("/increase_time", methods=["POST"])
-def increase_time():
-    """Increases the block timestamp offset"""
+async def increase_time():
+    """Increases the block timestamp offset and generate block"""
     request_dict = request.json or {}
     time_s = extract_positive(request_dict, "time")
 
     state.starknet_wrapper.increase_block_time(time_s)
+    await state.starknet_wrapper.update_pending_state()
+    await state.starknet_wrapper.update_pending_block()
+    block = await state.starknet_wrapper.generate_latest_block()
 
-    return jsonify({"timestamp_increased_by": time_s})
+    return jsonify(
+        {"timestamp_increased_by": time_s, "block_hash": hex(block.block_hash)}
+    )
 
 
 @base.route("/set_time", methods=["POST"])
-def set_time():
-    """Sets the block timestamp offset"""
+async def set_time():
+    """Sets the block timestamp offset and generate block"""
```

Comment:
```suggestion
    """Sets the block timestamp offset and generates a block"""
```

Reply:
fixed in bedd89c410df8d46c1a98138aa4cdbb9b53f69eb